README.md

```diff
@@ -13,8 +13,8 @@ This is a fabric mod that introduces a brand new **Vulkan** based voxel renderin
 [![Demostration Video](http://img.youtube.com/vi/sbr7UxcAmOE/0.jpg)](https://youtu.be/sbr7UxcAmOE)
 
 ## FAQ
-
-- Remember to check the [FAQ](./faq.md) we wrote before asking for support!
+<!-- REMEMBER TO CHANGE THE LINK ONCE THE PR IS READY !!!! -->
```

Comment:
Yep @xCollateral Tell me when you think the PR is ready, cuz we'll have to change `SpidFightFR` by `xCollateral`, once the Wiki is ready.